cni/network/network.go

```diff
@@ -301,6 +295,30 @@
 	}
 }
 
+func acquireLockForStore(config *common.PluginConfig, plugin *netPlugin) error {
+	var err error
+	if err = plugin.InitializeKeyValueStore(config); err != nil {
+		log.Errorf("Failed to initialize key-value store of network plugin, err:%v.\n", err)
+
+		if isSafe, _ := plugin.IsSafeToRemoveLock(azureCniName); isSafe {
+			log.Printf("[CNI] Removing lock file as process holding lock exited")
+			if err = plugin.UninitializeKeyValueStore(true); err != nil {
+				log.Errorf("Failed to uninitialize key-value store of network plugin, err:%v.\n", err)
+			}
+		}
+	}
+
+	return err
+}
+
+func releaseLockForStore(plugin *netPlugin) error {
```

Comment:
Can we add a comment that we are releasing the lock with pid?

Reply:
added log..pid will be part of every log..so we can figure out from that